src/components/theme.ts

```diff
@@ -52,7 +52,7 @@ const colors = {
   activeShading: "rgba(27, 26, 33, 0.16)",
 };
 
-const breakpoints: string[] = ["319px", "424px", "767px", "1023px"];
+const breakpoints: string[] = ["319px", "424px", "767px", "1023px", "1339"];
```

Comment:
that final breakpoint looks as if its missing a px

Reply:
Fixed, with a minor edit to the `left` position of the last value of the props array for each icon.

Reply:
Very minor, but could we put 
```
button {
  font-family: ....
}
```

in index.css

Reply:
This is a merge conflict issue =( 

I tried updating the index.css to the latest version (as seen in [PR #49](https://github.com/Jbarget/oxymore/pull/49/files)) buuut it wouldn't let me because the branch has been open a while and was created off an old version of `develop`. 

If we can keep the index.css as is on this one, we can merge in the changes from PR#49.